public/javascripts/SVLabel/src/SVLabel/Main.js

```diff
@@ -436,6 +436,9 @@ function Main (params) {
                 svl.neighborhoodModel.setNeighborhoodCompleteAcrossAllUsers();
             }
 
+            // Check make all links on the panorama clickable
+            svl.map.makeLinksClickable();
```

Comment:
I remember us trying to add this because we thought the code just wasn't being called since we weren't seeing any logging, but did this actually end up being necessary? I expected it to just work without adding this extra code.